test/test_basic.py

```diff
@@ -16,12 +16,36 @@
 class IntegrationTest(unittest.TestCase):
 
     def setUp(self):
-        subprocess.check_call(["make", "run"], cwd=projroot)
-        # Wait until the appservice container is up
-        self.ssl_3scale = ssl.create_default_context(cafile=os.path.join(projroot, '3scale', 'certs', 'ca.crt'))
-        self.request('https://localhost:8443/api/webconsole/v1/ping', retries=20)
+        try:
+            subprocess.check_call(["make", "run"], cwd=projroot)
+            # Wait until the appservice container is up
+            self.ssl_3scale = ssl.create_default_context(cafile=os.path.join(projroot, '3scale', 'certs', 'ca.crt'))
+            self.request('https://localhost:8443/api/webconsole/v1/ping', retries=20)
+        except (subprocess.CalledProcessError, AssertionError, IOError, OSError):
+            self.dumpLogs()
+            raise
+
+    def dumpLogs(self):
+        ids = subprocess.check_output(['podman', 'ps', '--all', '--quiet', '--filter', 'pod=webconsoleapp'],
```

Comment:
FTR, I'd really like to filter on `network=consoledot`, which ought to catch the session containers, too. But that ignores the app pod unfortunately (that feels like a podman bug).

I'll change it to do both queries and merge them.

Reply:
Done.